src/scss/includes/components/panel.scss

```diff
@@ -87,6 +87,7 @@ $bottom-margin: 40px;
     }
 
     .panel-drawer {
+      width: 100%;
       flex-shrink: 0;
     }
```

Comment:
Isn't that change in conflict with recent changes in https://github.com/QwantResearch/erdapfel/pull/815
where width is set explicitly at 50% on line 31 ?

Reply:
> I'm not sur in which case setting width to 50% is needed ?

@G-Ray it's a usability feature, related to the close button, so a user who missed the touch on the button by one pixel doesn't risk to minimize the panel.
I think the right way to fix this 2-lines issue is to move the title outside of the drawer.